src/property-filter/token.tsx

```diff
@@ -92,7 +97,12 @@ export const TokenButton = ({
         const formattedToken = i18nStrings.formatToken(token);
         return {
           content: (
-            <span className={styles['token-trigger']}>
+            <span
+              className={clsx(styles['token-trigger'], analyticsSelectors['token-trigger'])}
+              {...getAnalyticsMetadataAttribute({
+                action: 'editStart',
```

Comment:
We already support grouped tokens internally. For grouped tokens, the edit operation originates from a dedicated button. I think this action should be defined in the filtering-token/index.tsx

Reply:
Since grouped tokens are not in prod today, can I move that logic at a later stage?

Reply:
Implemented the change. I still think it is not very convenient, because it requires the addition of an unnecessary span, but I am fine with that